python/hidet/testing/models/llama.py

```diff
@@ -357,8 +357,8 @@ def convert_model(hf_model: torch.nn.Module, dtype=hidet.float16, device='cuda')
 
 def build_flow_graph(model, batch_size=1, device='cuda', dtype='float16'):
     config = model.config
-    input_ids = hidet.symbol([batch_size, "seq_length"], dtype=hidet.int32, device=device)
-    position_ids = hidet.symbol([batch_size, config.max_position_embeddings], dtype=hidet.int32, device=device)
+    input_ids = hidet.symbol([batch_size, "seq_length"], dtype=hidet.int64, device=device)
```

Comment:
I know there is a error to use "hidet.int32" here, but is there a way to fix that error directly instead of using "int64"?

Reply:
fixed by casting the model output to int32

Reply:
I see. From https://github.com/hidet-org/hidet/blob/f937ad23ef28558d205276f7588cca805377370a/python/hidet/graph/ops/reduce/reduce.py#L209, we seems force the output of arg reduce to `int64` (seems to be consistent with pytorch, #111). 

Might be better to add a parameter to all arg reduce operator (e.g., `def argmin(..., out_dtype='int64')`) to allow we use other data types.